frontend/packages/ceph-storage-plugin/src/__tests__/restore-pvc-modal.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { ShallowWrapper, shallow } from 'enzyme';
+import { TextInput, FormGroup, Grid } from '@patternfly/react-core';
+
+import { ModalTitle, ModalSubmitFooter } from '@console/internal/components/factory';
+
+import {
+  RestorePVCModalProps,
+  RestorePVCModal,
+} from '../components/modals/restore-pvc-modal/restore-pvc-modal';
+import { snapshotData } from '../__mocks__/volume-snapshot-data';
+import { VolumeSnapshotModal } from '../components/modals/volume-snapshot-modal/volume-snapshot-modal';
+
+import Spy = jasmine.Spy;
+
+describe(VolumeSnapshotModal.name, () => {
```

Comment:
This unit test works with `RestorePVCModal` (?)

```suggestion
describe(RestorePVCModal.name, () => {
```